pymodels/SI_V25_04/lattice.py

```diff
@@ -761,11 +760,10 @@ def set_vacuum_chamber(
     scrapv_vchamber = [-0.0145, 0.0145, -0.00475, 0.00475]
     scraph_vchamber = [-0.012, 0.012, -0.012, 0.012]
 
-    ida_vchamber = [-0.012, 0.012, -0.004, 0.004]
-    idb_vchamber = [-0.004, 0.004, -0.00225, 0.00225]
-    idp_vchamber = ida_vchamber
-    if optics_mode.startswith('S05'):
-        idp_vchamber = idb_vchamber
+    # BSC v defined by IVU gap + copper foil = 2 mm @ 1 m from ss center
+    # BSC h defined by vaccum chamber - no limitant
```

Comment:
Suggestion:

BSC v: defined by IVU magnetic gap + copper foil, 2 mm @ 1m from low beta ss center
BSC h: defined by standard circular vacuum chamber of 12-mm diameter (@ xx meters from high beta ss center)